package day37.tasks.Exceptions;

public class ErrorEx {
    public static void main(String[] args) {

        try{
            //your code
        }catch(Error er){

        }

    }
}